### PR TITLE
Add Recipe for AutoKeras

### PR DESCRIPTION
**Pull request by Anselmoo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Anselmoo** [comment]
@conda-forge-admin, please rerender

**Comment by Anselmoo** [comment]
@conda-forge/help-python

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ build:
   # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
```
Comment (dopplershift) [inline recipes/example/meta.yaml:33]: Please don't modify the example recipe. Can you revert the changes here?

File: recipes/autokeras/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (dopplershift) [inline recipes/autokeras/LICENSE:1]: The license is already included in the source tarball so you don't need to (and shouldn't) include it in the recipe. Please go ahead and remove.

File: recipes/autokeras/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  number: 0
+  skip: True  # [win or py<37]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"  # [not win]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (dopplershift) [inline recipes/autokeras/meta.yaml:22]: For noarch:
```suggestion
    - python >=3.7
    - pip
  run:
    - python >=3.7
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
A few minor changes, the biggest of which is to make this noarch (which drastically cuts down the number of builds). We can ignore any CI build failures on Windows due to missing dependencies (since by default noarch packages are only *built* on Linux).

File: recipes/example/meta.yaml (hunk 1 above)
Comment (Anselmoo) [reply recipes/example/meta.yaml:33]: I will take care!

File: recipes/autokeras/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autokeras-{{ version }}.tar.gz
+  sha256: 3d0f09629abb9b98f9059e3c8b9d39b4acd7f6dcde6d0584146a02035159141a
+
+build:
+  number: 0
+  skip: True  # [win or py<37]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"  # [not win]
```
[the remark this answers is not in the document]
Comment (Anselmoo) [reply recipes/autokeras/meta.yaml:15]: Unfortunately, this is not possible due to https://github.com/conda-forge/tensorflow-feedstock/issues/124
**Windows-Builds** will crash... sorry I made a force push to overwrite the failed **Windows-Builds**

File: recipes/autokeras/meta.yaml (hunk 3 above)
Comment (Anselmoo) [reply recipes/autokeras/meta.yaml:22]: See the comment above, please.

File: recipes/autokeras/LICENSE (hunk 2 above)
Comment (Anselmoo) [reply recipes/autokeras/LICENSE]: Is removed

File: recipes/example/meta.yaml (hunk 1 above)
Comment (Anselmoo) [reply recipes/example/meta.yaml]: I have to add the whitespace manually.

**Review by Anselmoo — COMMENTED** [review]
`meta.yaml` is reformat

File: recipes/autokeras/meta.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (dopplershift) [reply recipes/autokeras/meta.yaml:15]: Right, we can ignore the failed Windows builds this results in because it would be a noarch packages. noarch packages are packages that will work on every platform, but (by default) are only built on Linux. So the build would succeed. The package wouldn't be *installable* on Windows until we get new builds for tensorflow on Windows.

The benefit of noarch is it drastically cuts down on the number of builds (makes it easier when new versions of Python appear).

[the remark this answers is not in the document]
Comment (Anselmoo) [reply recipes/autokeras/meta.yaml:15]: Thx, @dopplershift , for clarifying this. I thought a failed build would not be legal.

File: recipes/autokeras/meta.yaml (hunk 3 above)
Comment (Anselmoo) [reply recipes/autokeras/meta.yaml:22]: Added `python >=3.7`

**Review by dopplershift — APPROVED** [review]

**Comment by dopplershift** [comment]
Ignoring noarch Windows build failure due to missing dependency.
